README.md

```diff
@@ -4,45 +4,53 @@ This repository contains scripts to build server binaries and binaries for mods
 
 ## Supported Mods
 
-**Maintained** - List of supported mods in Teeworlds 0.6.x:
-
-- DDNet - [Source](https://github.com/ddnet/ddnet) | [Website](https://ddnet.tw/)
-- InfClass - [Source](https://github.com/InfectionDust/teeworlds-infclassr)
-- MMOTee - [Source](https://github.com/kurosio/Teeworlds-Mmotee-Old)
-- ZombPanic - [Source](https://github.com/teeframe/zombpanic)
-
-**Unmaintained** - List of supported mods in Teeworlds 0.6.x:
-
-- Bomb Tag - [Source](https://github.com/unique-clan/bomb)
-- Flagball - [Source](https://github.com/teeframe/flagball)
-- Football - [Source](https://github.com/unique-clan/football)
-- HMH-Monster - [Source](https://github.com/teeframe/hmh-monster)
-- Hunter - [Source](https://github.com/yangfl/teeworlds-hunter)
-- Killing Floor - [Source](https://github.com/Siile/KillingFloor)
-- TeeSmash - [Source](https://github.com/timazuki/TeeSmash)
-- TeeWare - [Source](https://github.com/headshot2017/teeware-mod)
-- zCatch - [Source](https://github.com/ddnet/zcatch)
-
-**Maintained** - List of supported mods in Teeworlds 0.7.x:
-
-- MMOTee - [Source](https://github.com/MrCosmo666/Teeworlds-MRPG) ***Requires Custom Client**
-
-**Unmaintained** - List of supported mods in Teeworlds 0.7.x:
-
-- Nodes - [Source](https://github.com/teeworldsnetwork/nodes) | [Website](https://nodes.teeworlds.dev/) ***Requires Custom Client**
+| **Maintained** | List of supported mods in Teeworlds 0.6.x |
```

Comment:
The idea of converting from list to table is to segment the names and links into columns instead of a prose format. Something like:

| Name | Website | Source | Forum Post | Succeeds the mods  (suggestion) |

The same goes for the binary mods.